src/js/relay.firefox.com/get_profile_data.js

```diff
@@ -222,238 +209,53 @@
 
       browser.storage.local.set({ relayAddresses: localCopyOfServerRelayAddresses });
     } else {
-      // Scrape alias data from Profile page (Local)
-
-      // await browser.runtime.sendMessage({
-      //   method: "displayBrowserActionBadge",
-      // });
+      // Since the React version handles local storage of
+      // label data by itself, we don't need to add our own listeners,
+      // and we can just fetch label data from the API.
+      const { relayAddresses: existingLocalStorageRelayAddresses } = await browser.storage.local.get(
+        "relayAddresses"
+      );
 
-      // Scrape data from /accounts/profile/ page
-      for (const aliasCard of dashboardRelayAliasCards) {
-        // Add the description (previoulsy domain) note from the addon storage to the page
+      // await browser.storage.local.set({ relayAddresses: {}} });
 
-        const aliasCardData = aliasCard.dataset;
-        const aliasId = parseInt(aliasCardData.relayAddressId, 10);
-        const addonRelayAddress = addonRelayAddresses.relayAddresses.filter(
-          (address) => address.id == aliasId
-        )[0];
 
-        const defaultAliasLabelText = browser.i18n.getMessage(
-          "profilePageDefaulAliasLabelText"
-        );
```

Comment:
Removed legacy code that was necessary for the Django frontend.

Reply:
Would it be possible to rebase on the other PR, or at least split the removal of legacy code into a separate commit? Then it's easier to see the scope and approach of the bugfix.